provider/vpod_store.go

```
@@ -80,6 +81,7 @@ func (r *VPodStore) CheckContainerStatusNeedSync(bizStatusData model.BizStatusDa
 	r.Lock()
 	defer r.Unlock()
 
+	fmt.Println(fmt.Sprintf("CheckContainerStatusNeedSync: %v, %v", bizStatusData, r.podKeyToPod))
```

golangci-lint (1.62.2)

S1038: should use fmt.Printf instead of fmt.Println(fmt.Sprintf(...)) (but don't forget the newline)

(gosimple)